cmd/system-probe/modules/network_tracer.go

```diff
@@ -91,6 +91,18 @@ func (nt *networkTracer) Register(httpMux *module.Router) error {
 		logRequests(id, count, len(cs.Conns), start)
 	}))
 
+	httpMux.HandleFunc("/register", utils.WithConcurrencyLimit(utils.DefaultMaxConcurrentRequests, func(w http.ResponseWriter, req *http.Request) {
```

Comment:
In the interest of further module isolation, could we have the path be `/network_tracer/register`?